packages/terra-table/src/subcomponents/Cell.jsx

```diff
@@ -136,6 +136,12 @@ const propTypes = {
    * The color to be used for highlighting a column.
    */
   columnHighlightColor: PropTypes.oneOf(Object.values(ColumnHighlightColor)),
+
+  /**
+   * Enables row selection capabilities for the table.
+   * Use 'single' for single row selection and 'multiple' for multi-row selection.
+   */
+  rowSelectionMode: PropTypes.string,
```

Comment:
Should this be private? Since the rowSelectionMode is set by user on the Row and you are just passing the value down here internally.

Reply:
Not needed I guess since there are other non-private props as well (like height) similar to this passed down internally to cell. But anyways marked private: https://github.com/cerner/terra-framework/pull/2088/commits/6ecfb18835da3125978259937907002382fac3cf